source/includes/administration/_pricings.md

```diff
@@ -231,3 +231,133 @@ Attributes | &nbsp;
 `pricingProducts.cogs`<br/>*double* | The Cost Of Goods Sold (COGS) of the product.
 `pricingProducts.currency`<br/>*string* | The currency of the pricing.
+
+<!-------------------- CREATE PRICING -------------------->
+#### Create pricing
+
+`POST /pricings`
+
+Create a new pricing.
+
+```shell
+# Creates a new pricing
+curl -X POST "https://cloudmc_endpoint/rest/pricings" \
+   -H "MC-Api-Key: your_api_key"
+```
+
+> Request body example(s):
+
+```js
+{
+	"organization": {
+		"id": "1d37a999-6ac6-4193-acb2-49df5c43ff31"
+	},
+	"productCatalogs": [
+		{
+			"id": "b6c89a59-34bf-4be4-a9bf-4414de84f5f7"
+		}
+	],
+	"name": {
+		"en": "name",
+		"fr": "nom",
+		"es": "nombre"
+	},
+	"description": {
+		"en": "desc_en",
+		"fr": "desc_fr",
+		"es": "desc_es"
+	},
+	"effectiveDate": "2020-05-06T00:00:00.000Z",
+	"pricingProducts": [
+		{
+			"product": {
+				"id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3"
+			},
+			"cogs": 4,
+			"unitPrice": 5,
+			"currency": "CAD"
+		},
+		{
+			"product": {
+				"id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3"
+			},
+			"cogs": 4,
+			"unitPrice": 5,
+			"currency": "USD"
+		}
+	],
+	"supportedCurrencies": [
+		"CAD",
+		"USD"
+	]
+}
+```
+> The above command(s) return(s) JSON structured like this:
+
+```js
+{
+  "data": {
+    "supportedCurrencies": [
+      "CAD",
+      "USD"
+    ],
+    "productCatalogs": [
+      {
+        "id": "b6c89a59-34bf-4be4-a9bf-4414de84f5f7"
+      }
+    ],
+    "organization": {
+      "id": "1d37a999-6ac6-4193-acb2-49df5c43ff31"
+    },
+    "pricingProducts": [
+      {
+        "unitPrice": 5,
+        "product": {
+          "id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3",
+        },
+        "cogs": 4,
+        "currency": "CAD",
+        "id": "45301951-59cb-461e-b0d0-85c46a7abb69"
+      },
+      {
+        "unitPrice": 5,
+        "product": {
+          "id": "c14daee2-4678-4710-b9af-fc26fbd3c7f3",
+        },
+        "cogs": 4,
+        "currency": "USD",
+        "id": "30ec5361-0044-44d5-a8b1-68dca07d274e"
+      }
+    ],
+    "name": {
+      "en": "name",
+      "fr": "nom",
+      "es": "nombre"
+    },
+    "changes": [],
+    "description": {
+      "en": "desc_en",
+      "fr": "desc_fr",
+      "es": "desc_es"
+    },
+    "id": "c4860b7d-ad41-40a8-b2a2-6f96be87c8b0",
+    "effectiveDate": "2020-05-06T00:00:00Z"
+  }
+}
+```
+
+Required | &nbsp;
+------- | -----------
+`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
+`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
+`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
+`name` <br/>*Map[string,string]* | A map of language keys to a name in the specified language. 
+`description` <br/>*Map[string,string]* | A map of language language keys to the description in the specified language.
```

Comment:
same as for name

Reply:
https://github.com/cloudops/cloudmc-api-docs/pull/142/commits/5a17a84d28c2a86b44de607482ee752c52102de4